src/firefly/js/ui/dynamic/EmbeddedPositionSearchPanel.jsx

```diff
@@ -155,32 +164,136 @@ export function EmbeddedPositionSearchPanel({
+                      if (!isSearchPanel) setIsHovered(false);
+                      else setIsHovered(true);
+             }}
+            style={{display: 'flex', flexDirection: 'column', alignItems: 'center', alignSelf: 'stretch', height:'100%',
+                         paddingBottom:insetSpacial?0:20, position: 'relative'}}>
+            <HiPSTargetView
+                {...{
                 hipsUrl, centerPt:initCenterPt, hipsFOVInDeg, mocList, coordinateSys, sRegion, plotId,
                 minSize: minValue, maxSize: maxValue, toolbarHelpId,
                 whichOverlay: doGetConeAreaOp(), setWhichOverlay: doToggle ? setConeAreaOp : undefined,
                 targetKey, sizeKey, polygonKey, sx: {minHeight: 300, alignSelf: 'stretch', flexGrow:1}
             }}/>
-            <Sheet {...{className:`FFepsp-content ${insetSpacial ? 'inset' : ''}`,
-                sx: (theme) => (
-                    {
-                        alignItems: 'center',
-                        alignSelf: 'stretch',
-                        borderRadius: '5px 5px 2px 2px',
-                        border: `3px solid ${theme.vars.palette['neutral']?.softActiveBg}`,
-                        position: 'absolute',
-                        px: 1/2,
-                        bottom: '1.5rem',
-                        left: 3,
-                        ...slotProps?.searchRoot?.sx
-                    })
-
-            }}>
-                {wrappedInternals}
+            <Sheet
+                onMouseEnter={() => {
+                    setIsHovered(true);
+                    setIsSearchPanel(true);
+                }}
+                onMouseLeave={() => {
+                    setIsSearchPanel(false);
+                }}
+                {...{className:`FFepsp-content ${insetSpacial ? 'inset' : ''}`,
+                    sx: (theme) => (
+                        {
+                            alignItems: 'center',
+                            alignSelf: 'stretch',
+                            borderRadius: '5px 5px 2px 2px',
+                            border: `3px solid ${theme.vars.palette['neutral']?.softActiveBg}`,
+                            position: 'absolute',
+                            px: 1/2,
+                            bottom: '1.5rem',
+                            maxWidth: '50em',
+                            left: 3,
+                            opacity: isHovered ? '100%' : '40%',
+                            ...slotProps?.searchRoot?.sx
+                        })
+                }}>
+                <CollapsibleGroup
+                    sx={{borderWidth: '0px',
+                        '& .MuiAccordionSummary-root': {
+                            paddingBlockStart: '0.5rem',
+                            paddingBlockEnd: '0.5rem',
+                            minBlockSize: '1rem',
+                            '&.Mui-expanded': {
+                                height: '1rem '
+                            }
```

Comment:
I don't think these add any value except for additional padding which, in my opinion, is not needed.  It however makes it really hard for someone else to modify months from now.

Reply:
Okay so the reasoning behind this one is interesting. From what I can tell Joy/MUI's AccordionSummary assigns the following values: 

``` 
paddingBlockStart: var(--ListItem-paddingY)
paddingBlockEnd: var(--ListItem-paddingY)
minBlockSize: var(--ListItem-minHeight) 
```

which are inherited from the default `JoyAccordionGroup` as: 
``` 
--ListItem-minHeight: 2.25rem;
--ListItem-paddingY: 0.25rem;
--ListItem-paddingX: 0.75rem; 
```

Now I could potentially set these in ThemeSetup.js if we wanted to use these values (the ones I overrode) elsewhere too, but it actually looks good in other places, for example this one from Firefly Image tab (ImageSelect.jsx). 

![Screenshot 2024-05-09 at 7 29 14 PM](https://github.com/Caltech-IPAC/firefly/assets/110492183/dbdc4146-ff2e-4ea2-9d6c-be143ee6118f)
 as the default. 

Now the same (without me overriding the values) looks like this on Euclid (compare it with the test build): 

![Screenshot 2024-05-09 at 7 33 39 PM](https://github.com/Caltech-IPAC/firefly/assets/110492183/67881d07-5149-4ebe-aa8b-70bfe316e10d)

which @robyww mentioned makes the header too big when the search panel is open. 

So in order to make it smaller, I added this extra bit of styling. It's possible there may be simpler way to set the height, but I tried a few things and unless I overrode (or unchecked) this default style, my desired height for the header/summary was not respected. 
 
Anyway, if we decide to keep this, I can add a small comment explaining this choice for future developer to understand at first glance.